astropy/_erfa/core.py.templ

```diff
@@ -32,7 +32,7 @@ import warnings
 
 import numpy
 
-from astropy.utils.exceptions import AstropyUserWarning
+from astropy.utils.exceptions import ErfaError, ErfaWarning
```

Comment:
```suggestion
# we import these exceptions from astropy locations instead of defining them
# in this file because otherwise there are circular dependencies
from astropy.utils.exceptions import ErfaError, ErfaWarning
```